File: src/com/lonelybytes/swiftlint/annotator/InitialInfo.java
package com.lonelybytes.swiftlint.annotator;

public class InitialInfo {
    String filePath;
    boolean shouldProcess;
    int documentLineCount; // TODO: This value can be wrong :(

    InitialInfo(String aFilePath, boolean aShouldProcess, int aDocumentLineCount) {
        filePath = aFilePath;
        shouldProcess = aShouldProcess;
        documentLineCount = aDocumentLineCount;
    }
}

File: src/com/lonelybytes/swiftlint/annotator/InitialInfo.kt
package com.lonelybytes.swiftlint.annotator

import com.intellij.openapi.editor.Document
import com.intellij.psi.PsiFile

class InitialInfo internal constructor(
    val file: PsiFile,
    val path: String,
    val document: Document?,
    val shouldProcess: Boolean,
    val runDirectoryFromConfigPath: String?
)